src/brace_cleanup.cpp

```diff
@@ -695,7 +696,8 @@ static void parse_cleanup(parse_frame_t *frm, chunk_t *pc)
    {
       if (!cpd.consumed)
       {
-         if (!cpd.unc_off_used)
+         size_t file_pp_level = ifdef_over_whole_file() ? 1 : 0;
+         if (!cpd.unc_off_used && pc->pp_level == file_pp_level)
```

Comment:
While this is not ideal is a lot better than before as it covers two most used patterns.